app/controllers/admin/users_controller.rb

```diff
@@ -55,7 +55,7 @@ def export_data
       user = User.find(params[:id])
       send_to_admin = params[:send_to_admin].to_boolean
       if send_to_admin
-        email = SiteConfig.email_addresses[:default]
+        email = SiteConfig.email_addresses[:contact]
```

Comment:
I thought that the admin export for data should go to the contact email, not "nonreply" one. Willing to revert that if we should keep the status quo